graphene/types/schema.py

```diff
@@ -452,6 +452,9 @@ def __getattr__(self, type_name):
             return _type.graphene_type
         return _type
 
+    def get_type(self, name):
+        return self.graphql_schema.type_map.get(name)
```

Comment:
This will actually return a `GraphQLObjectType` of type `GrapheneObjectType` from `graphql-core`, not a graphene `ObjectType`. These two are fundamentally different classes. That's the reason why your test is failing. You'd need to return the Graphene type here.

Reply:
@erikwrede Thanks, I tried to update the change based on your feedback. How's that look?

Reply:
although i'm not sure entirely, as now i get:

```
  File "/home/circleci/project/venv_3_9_10/lib/python3.9/site-packages/graphene/relay/node.py", line 116, in get_node_from_global_id
    graphene_type = graphene_type.graphene_type
```